### DISCUSSION TITLE
General information

### DISCUSSION THREAD

**Discussion by jasperdekeuk** [simatic-ax/.discussions#11] [Q&A] [answered]
Hi,

Very much looking forward to this project!
But unfortunately it is not very clear what the current status is.
I can't signup (probably due to limited sales release) and have no idea how to get started or where to get more info.

Would it be possible to maybe include, in the general readme, following topics:

- Goals
- Status
- Getting started

A bit similar to [Carbon Language Project](https://github.com/carbon-language/carbon-lang#project-status)

The info given on SIOS can maybe be implemented here? (or provide the link)
[What is SIMATIC AX](https://support.industry.siemens.com/cs/document/109815017/what-is-simatic-ax-where-to-find-information-and-how-you-can-get-support-?dti=0&lc=en-AF)

**Comment by sjuergen** [comment, marked as answer]
Hi @jasperdekeuk thanks for your valuable feedback, after the holidays, I'll discuss your proposals with my colleagues. And you're right: due to the limited sales release, you can't signup directly. For now, the official way to get more information is to contact your regional Siemens contact. See also:

![image](https://user-images.githubusercontent.com/49430012/210011114-d40a0846-b9b0-40ea-b0e5-e43484e64229.png)

Let's see, what we can improve...

**Reply by drexxi** [reply]
Hello @sjuergen,

I am also interested in a test.
I have made a support request if i can get access.

I use TIA portal since version 10.5,
I also work a lot with the TIA Openess API,
and have developed myself a functional engineering tool for TIA and Unified,
so of course I would be happy to have a library management in Git.

I hope I get access.

**Reply by sjuergen** [reply]
Hi @drexxi,

thank you for your interest in SIMATIC AX. Did you made the request at your regional Siemens contact?

Regards
Juergen

**Reply by drexxi** [reply]
Yes we have done that, 
but the approach is still wrong for me.
The product should rather be distributed free of charge in the beta phase to get as many people as possible to taste it and collect impressions.
But that is just my opinion.

**Reply by MelHuber** [reply]
@drexxi great to hear that you already reached out your Siemens contact! We are aware that this go-2-market strategy is not suitable for everybody. Netherless we highly appreciate your feedback and will consider it regarding further steps.

**Comment by ahuca** [comment]
Hi @sjuergen,

On the same topic of getting more information and having the interest to try out Simatic AX. Is it possible to do so as an _individual hobbyist_? Does the same process apply in this case (contacting the regional Siemens support).

Regards,
Giang

**Reply by drexxi** [reply]
At Siemens, you have to pay for everything:
Knowledge, licenses, software...
that will be the downfall at some point.
I would rather choose a product that I can try for free and get knowledge about before I use it in projects.
Most software developers will think like that.

**Reply by MelHuber** [reply]
@ahuca Thanks a lot for your interest. We just started with a limited sales release focusing on our TIAX use case - the collaboration of TIA Portal V18 and SIMATIC AX. Currently the product is just available for selected countries within Europe. For now, individual hobbist can not get access. Please be patient. There is more coming soon! 😉

**Reply by ahuca** [reply]
@MelHuber thank you for the info. The potential of AX is looking very excited. I'll be looking forward to the global release!
